pages/en/lb4/LoopBack-3.x.md

```diff
@@ -75,7 +77,38 @@ LoopBack v4 is moving to the well-known Model-(View-)Controller pattern, where t
 | Tooling               | loopback-cli and API Connect UI             | (TBA)            |
 
 
+## What's new and exciting in LoopBack 4
+
+LoopBack 4 is envisioned with new and exciting ideas highlighted but not limited to as follows. 
+
+- Leverage TypeScript for better code quality and productivity
+- Unify and simplify the asynchronous programming model/style around Promise and Async/Await
+- Implement an IoC Container with Dependency Injection for better visibility, extensibility and composability
+- Introduce Component as packaging model for extensions that can be plugged into LoopBack 4 applications 
+- Make everything else as components, such as REST, Authentication, and Authorization
+- Divide the responsibilities of LoopBack models into
+  - Controllers - to handle incoming API requests
+  - Repositories - to provide access to data stores
+  - Models - to define schemas for business objects
+  - Services - to interact with existing REST APIs, SOAP WebServices, and other form of services/microservices
+- Refactor the ORM into separate modules for different concerns 
+
 ## What's in the beta release
 
+The beta release is the first milestone of the LoopBack 4 journey. It's in no way meant to be functionally complete or ready for prime time. You should see it as a preview of what's coming. But we do have some building blocks starting to take shape, such as:
+
+1. A new `@loopback/context` module that implements an IoC container with dependency injection
+2. A new `@loopback/core` module that defines core artifacts such as application and component
+3. A `@loopback/rest` component that provides top-down REST API mapping using OpenAPI/Swagger specs and controllers
+4. A `@loopback/authentication` component to provide infrastructure to integrate with authentication providers
+5. An experimental `@loopback/repository` module to define repository interfaces and provide a reference implementation on top of legacy `loopback-datasource-juggler` and connectors
+6. Examples and tutorials
+
+The primary target audience of the beta release is extension developers. Please check out the hello-extension.
+
+API developers can start to have a sneak peek of the LoopBack 4 applications even the CLI is not there. See hello-world, ...   
```

Comment:
This is incomplete.... Maybe something like:
"The initial beta release provides a preview for API developers.  Currently, the LoopBack CLI doesn't hyet support LoopBack 4, but it will eventually."